oshdb-api/src/main/java/org/heigit/ohsome/oshdb/api/mapreducer/MapAggregator.java

```diff
@@ -748,22 +747,8 @@ public void forEach(SerializableBiConsumer<U, List<X>> action) throws Exception
    */
   @Contract(pure = true)
   public Stream<Entry<U, X>> stream() throws Exception {
-    return this.mapReducer.stream().map(d -> new Entry<U, X>() {
-      @Override
-      public U getKey() {
-        return d.getKey();
-      }
-
-      @Override
-      public X getValue() {
-        return d.getValue();
-      }
-
-      @Override
-      public X setValue(X value) {
-        throw new RuntimeException("cannot modify the value of this entry");
-      }
-    });
```

Comment:
I replaced the anonym class with AbstractMap.SimpleImmutableEntry, I thought this makes sense in this context.